### DISCUSSION TITLE
Made max message size configurable in config.yaml, as a possible workaround for very large farms where reporting plot information exceeds the maximum message size.

### DISCUSSION THREAD

**Discussion by wanming851** [Chia-Network/chia-blockchain#8681] [Support]
Made max message size configurable in config.yaml, as a possible workaround for very large farms where reporting plot information exceeds the maximum message size.

Version 1.2.9
Where is it configured? I don't see it configured

**Comment by loppefaaret** [comment]
> You will need to add to config.yaml something like this
daemon_max_message_size: 60000000
or whatever size in bytes you want. I would add that near the top of the file where you see daemon_port:

see #8500 and #8061 sepcifically https://github.com/Chia-Network/chia-blockchain/issues/8061#issuecomment-921023270

**Comment by BrandtH22** [comment]
Closing this discussion as it is a duplicate of others previously linked (https://github.com/Chia-Network/chia-blockchain/pull/8500 and https://github.com/Chia-Network/chia-blockchain/issues/8061 specifically https://github.com/Chia-Network/chia-blockchain/issues/8061#issuecomment-921023270)
